schemas/interface.json

```diff
@@ -23,7 +23,8 @@
                         "description": {
                             "type": "string",
                             "minLength": 2
-                        }
+                        },
+                        "default": false            // we don't allow default values
```

Comment:
good question, I guess the current validation process (which includes patching the json with default values) will add defaults if not present in the data, so yes, it will be handled,but I don't know if that is desired behavior.

The better solution would be to remove all entries called `default` when loading the interface definition file, no matter their nesting level.